contracts/utils/ExpiringMetaTxForwarder.sol

```diff
@@ -22,19 +24,16 @@ import "./interfaces/IExpiringMetaTxForwarder.sol";
 /// owners to not care about if and when the other meta-tx is executed. The
 /// signer is responsible for not issuing signatures that may cause undesired
 /// race conditions.
-contract ExpiringMetaTxForwarder is EIP712, IExpiringMetaTxForwarder {
+contract ExpiringMetaTxForwarder is Context, EIP712, IExpiringMetaTxForwarder {
     using ECDSA for bytes32;
     using Address for address;
 
-    /// @notice If the meta-tx with hash is already executed
-    /// @dev This mapping is used to prevent identical meta-txes from being
-    /// executed. There are two issues with this: (1) The signer may actually
-    /// want to issue identical meta-txes (2) Setting a flag in the storage per
-    /// meta-tx is costly. Identical meta-tx validity can be disambiguated by
-    /// using a nonce that is specific to the signer–relayer pair, yet this is
-    /// decided against in favor of the simple UX that not having any kind of a
-    /// nonce provides.
-    mapping(bytes32 => bool) public override metaTxWithHashIsExecuted;
+    /// @notice If the meta-tx with hash is executed or nullified
+    /// @dev We track this on a meta-tx basis and not by using nonces to avoid
+    /// requiring users keep track of nonces
+    mapping(bytes32 => bool)
+        public
+        override metaTxWithHashIsExecutedOrNullified;
```

Comment:
maybe change to `metaTxWithHashIsProcessed`

Reply:
I think that's unnecessarily vague (in that what processing means is not clear). It could be `metaTxWithHashIsExecutable` but that implies that the meta tx with hash exists, which it may not, so this is the best option.